students/changhyeon/users/views.py

```diff
@@ -44,11 +47,20 @@ def post(self,request):
     try:
       email    = data["email"]
       password = data["password"]
+      user     = Users.objects.get(email=email)
 
-      if not Users.objects.filter(email = email, password = password).exists(): 
+      if not Users.objects.filter(email = email).exists(): 
         return JsonResponse({"message": "INVALID_USER"}, status = 401)
+      
+      valid_password = Users.objects.get(email=email).password.encode('utf-8')
 
-      return JsonResponse({"message": "SUCCESS"}, status = 200)
+      if not bcrypt.checkpw(password.encode('utf-8'), valid_password):
```

Comment:
이렇게 하셔서 불필요한 변수 지정은 줄여주세요!!
```suggestion
      if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
```

Reply:
확인했습니다!